.github/pull_request_template.md

```diff
@@ -9,16 +9,49 @@
 
 ## Issue type checklist
 <!--- What type of change are you submitting? Check the boxes that apply: -->
-- [ ] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
-- [ ] **New feature**: non-breaking change that adds functionality.
-- [ ] **Breaking change**: fix or feature that will cause current functionality to change.
+
+- [] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
+- [] **New feature**: non-breaking change that adds functionality.
+- [] **Breaking change**: fix or feature that will cause current functionality to change.
```

Comment:
I wonder if we shouldn't move "breaking" change out of this list into a separate bullet.

"This is a breaking change"

Reply:
@Falkicon do you have thoughts on this?

Reply:
Here's my case:

I cannot enumerate that something is both a fix and a feature. It seems that the sole purpose for "mark all that apply" is because "breaking change" exists here. Since breaking changes are *supplemental* to features and fixes, I propose we move it out as a separate bullet. We don't want to accept PR's that address multiple issues because if we need to back out a commit, we end up reverting multiple things. One issue per PR, or one fix/feature/chore per PR.

Breaking out "Breaking Change" allows us to say, "Select the type" instead of "Select all types". We send a unified message instead of multiple messages.